packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -559,21 +590,10 @@ export class OrchestratorService {
         if (status !== SyncStatus.PAUSED) {
             if (syncSchedule && syncSchedule?.info && syncSchedule?.info.futureActionTimes && syncSchedule?.info?.futureActionTimes?.length > 0) {
                 const futureRun = syncSchedule.info.futureActionTimes[0];
-                nextScheduledSyncAt = syncClient?.formatFutureRun(futureRun?.seconds?.toNumber() as number);
+                nextScheduledSyncAt = syncClient?.formatFutureRun(futureRun?.seconds?.toNumber() as number) || null;
             }
         }
 
-        return { schedule, latestJob, status, nextScheduledSyncAt };
-    }
-
-    private reportedStatus(
-        sync: Sync,
-        latestJob: SyncJob | null,
-        schedule: SyncSchedule | null,
-        status: SyncStatus,
-        nextScheduledSyncAt?: string | Date | null,
-        includeJobStatus = false
-    ) {
```

Comment:
this function was always used alongside the fetchSyncData. I merged them together into a syncStatus one